variables.tf

```
@@ -140,6 +140,12 @@ variable "deregistration_delay" {
   description = "The amount of time to wait in seconds before changing the state of a deregistering target to unused"
 }
 
+variable "slow_start" {
+  type        = number
+  default     = 0
```

So this doesn't show a change to people's resources, it might be better to default this to `null`

```suggestion
  default     = null
```